select.js

```diff
@@ -0,0 +1,14 @@
+define(function() {
+    var init = function() {
+        $('body').on('change', '.c-select', function() {
```

Comment:
In a stencil's JS, should we be using the `c-` class or should there be `js-` classes?